lib/mods/replace-methods.js

```diff
@@ -65,42 +65,42 @@ function getArgumentString(functionCall) {
 function replaceGlobalMethodCalls(node, imports) {
   if (nodeTypes.isCallExpression(node) &&
       isGlobal(node.getExpression())) {
-    let mod; let args; let type; let callee;
+    let moduleName; let args; let type; let callee; let propertyName;
 
     // property access methods
     if (nodeTypes.isPropertyAccessExpression(node.getExpression())) {
       const propertyAccess = node.getExpression();
-      name = propertyAccess.getSymbol().getEscapedName();
+      propertyName = propertyAccess.getName();
       type = 'method';
 
       let baseType = propertyAccess.getExpression().getType();
       const baseConstructSignature = baseType.getConstructSignatures()[0];
 
+      // the method is static if it has a constructSignature
       if (baseConstructSignature) {
         baseType = baseConstructSignature.getReturnType();
         type = 'static';
       }
 
-      mod = baseType.getSymbol().getEscapedName();
+      moduleName = baseType.getSymbol().getName();
       args = getArgumentString(propertyAccess.getParent());
```

Comment:
This line can move outside the ifs, to the bottom. In fact, maybe it can move inside `replaceNodeAndUpdateImports`?

Reply:
I'm gonna keep it out of `replaceNodeAndUpdateImports` for now because it only ever gets used in the method replacement mod, and args for the get-set mod are done in a different way. There is maybe a case to be made to make `getArgumentString` into a more generic method with a switch statement like some of the other methods, but I don't think it's necessary at the moment.